create_jenkins_job.py

```diff
@@ -172,7 +172,7 @@ def main(argv=None):
             job_name = job_name[0:-5]
         job_data['time_trigger_spec'] = '0 12 * * *'
         job_data['cmake_build_type'] = 'None'
-        job_data['ament_test_args_default'] = '--retest-until-fail 20'
+        job_data['ament_test_args_default'] = '--retest-until-fail 20 -LE linter'
```

Comment:
Sorry local aliases took care the `ctest-args` for me. Will update it, test it on the farm and resubmitted for review once it's actually ready